obtain.lua

```diff
@@ -0,0 +1,42 @@
+-- obtain.lua: arguments => [value, tokenLen, ttl]
+-- Obtain.lua try to set provided keys's with value and ttl if they do not exists.
+-- Keys can be overriden if they already exists and the correct value+tokenLen is provided. 
+
+local function pexpire(ttl)
+	-- Update keys ttls.
+	for _, key in ipairs(KEYS) do
+		redis.call("pexpire", key, ttl)
+	end
+end
+
+-- canOverrideLock check either or not the provided token match
+-- previously set lock's tokens.
+local function canOverrideKeys() 
+	local offset = tonumber(ARGV[2])
+
+	for i, key in ipairs(KEYS) do
+		if i % 2 == 1 then
```

Comment:
sorry, what ma I missing here? why every 2nd key?

Reply:
Oups, that's a silly mistake. This line was kept after I removed values from KEYS array.
I'll remove it.